### DISCUSSION TITLE
Question regarding automations

### DISCUSSION THREAD

**Discussion by Arturfrain** [netzero-labs/netzero#34] [General]
Hi, I have a question regarding automations. I have installed the App on IOS. I added my Tesla account data and my Tibber Auth. The dynamic Tibber Tarif was properly added to my Tesla account. I have a fixed export price which is very low in Germany. I would like to charge my Powerwall dependent on cheap grid price. 
Can I just set the time based control in the Tesla App relying on the Tarif plan which is controlled by net zero? Or do I need to set up extra automations in the net zero app?
I would like to understand the logic of the App. The Documentation is not detailed enough.

**Comment by zigam** [comment]
You do not need any extra automations, just Time-Based Control and automatic tariff updates. Time-Based Control is described in more detail here:
https://www.tesla.com/support/energy/powerwall/mobile-app/time-based-control-user-guide

It may take a few days to get familiar with how it operates, just know that it's only optimizing for savings, and that predicted solar generation and house usage will not always be correct, so don't expect perfect decisions at all times.

**Reply by Arturfrain** [reply]
Thank you very much for your help. I will test it and if some more questions arise I will write here if that is ok.
